test/macro-annot/run/kase/macro_kase_1.scala

```diff
@@ -69,7 +69,7 @@ object kaseMacro {
     def expand(annottees: List[c.Tree]): List[c.Tree] = {
       val cdef @ ClassDef(_, name, tparams, Template(_, _, cbody)) = annottees.head
       val primaryCtor = cbody.collect{ case ddef @ DefDef(_, termNames.CONSTRUCTOR, _, _, _, _) => ddef }.head
-      if (primaryCtor.vparamss.isEmpty) c.abort(c.enclosingPosition, "`kase` is not appplicable to classes without a parameter list")
+      if (primaryCtor.vparamss.isEmpty) c.abort(c.enclosingPosition, "`kase' is not applicable to classes without a parameter list")
```

Comment:
except I just noticed the quote went in the wrong direction. They prefer markdown style backticks now.

Reply:
but it's only a test...

Reply:
I didn't notice the change to the backtick. Not intentional. I'll watch for IntelliJ's "helping hand" in future. I'll revert this as part of a future PR.